lib/src/main/java/org/buffer/android/counterview/CounterView.kt

```diff
@@ -82,23 +83,39 @@ class CounterView : AppCompatTextView {
         }
         contentDescription = context.getString(R.string.character_count_description,
                 contentLength, counterMaxLength)
+        updateTextColor(contentLength)
```

Comment:
why do you need to update the text color when the text is changed?

Reply:
This is what was happening before I made my changes. I'm assuming it was put here so that when the character count changes to over/under the limit the color is updated accordingly.

Reply:
Ah ok, I can't see it in the previous commits of the project. It shouldn't be needed here as the color is being set elsewhere

Reply:
@hitherejoe I must be missing where it was set before. When I remove this, the color isn't changed when you type over the threshold. I'm also not seeing another place on master where the color is set in this case. Are you thinking about [here](https://github.com/bufferapp/CounterView/blob/master/lib/src/main/java/org/buffer/android/counterview/CounterView.kt#L85)?

Reply:
Ah yes, I missed that - no problem!